packages/admin/admin/src/table/TableDndOrder.tsx

```diff
@@ -104,27 +110,80 @@ function DndOrderRow<TRow extends IRow>(props: IDndOrderRowProps<TRow>) {
 
     return (
         <TableBodyRow ref={refRow} {...rowProps} style={{ opacity }}>
-            <TableCell ref={refDragHandle} className={classes.dragCell}>
-                <div className={classes.dragIconContainer}>{props.dragHandleIcon}</div>
-            </TableCell>
+            <DragCell ref={refDragHandle} {...slotProps?.dragCell} {...restProps}>
+                <DragItemContainer {...slotProps?.dragIconContainer}>{props.dragHandleIcon} </DragItemContainer>
+            </DragCell>
             <TableColumns columns={columns} row={row} />
         </TableBodyRow>
     );
 }
 
-interface TableDndOrderProps<TRow extends IRow> extends ITableProps<TRow> {
+const Root = styled(Table, {
+    name: "CometAdminTableDndOrder",
+    slot: "root",
+    overridesResolver(_, styles) {
+        return [styles.root];
+    },
+})();
+
+const DragCell = styled(TableCell, {
+    name: "CometAdminTableDndOrder",
+    slot: "dragCell",
+    overridesResolver(_, styles) {
+        return [styles.dragCell];
+    },
+})(css`
+    cursor: grab;
+    width: 20px;
+    padding-right: 0;
+`);
+
+const DragItemContainer = styled("div", {
+    name: "CometAdminTableDndOrder",
+    slot: "dragItemContainer",
+    overridesResolver(_, styles) {
+        return [styles.dragItemContainer];
+    },
+})(css`
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    :after {
+        // Increase clickable area to include the padding-left of the next cell.
+        content: "";
+        position: absolute;
+        z-index: 1;
+        top: 0;
+        right: -20;
```

Comment:
Unlike the previous object-syntax, the css-syntax requires units. 

```suggestion
        right: -20px;
```